epub33/rs/index.html

```diff
@@ -1594,9 +1595,16 @@ <h4>Initial containing block dimensions</h4>
 										<code>viewport</code>
 									<code>meta</code> tag for [=XHTML content documents=], as defined in <a
 										data-cite="epub-33#sec-fxl-icb-html">Expressing in HTML</a> [[epub-33]].</span>
-								<span id="confreq-fxl-rs-xhtml-icb" data-tests="#fxl-xhtml-icb">They MUST clip content
-									positioned outside of the ICB.</span>
+								<span id="confreq-fxl-rs-xhtml-icb" data-tests="#fxl-xhtml-icb,fxl-xhtml-icb_multi">They
+									MUST clip content positioned outside of the ICB.</span>
 							</p>
+							<p>If the <code>viewport</code>
+								<code>meta</code> does not contain a width or a height value, or if these values are
+								invalid, reading systems SHOULD consider these as having the values
+									<code>device-width</code> and <code>device-height</code>, respectively. Reading
+								systems MAY also use best effort in interpreting values that follow a deprecated syntax
```

Comment:
This will have to change as we can't normatively ask for best efforts... 😉 

It's not clear what it refers to, either, as the CSS module doesn't define a syntax, let alone a deprecated one.